chsdi/templates/htmlpopup/seilbahnenbundeskonzession.mako

```diff
@@ -0,0 +1,18 @@
+<%def name="anlage()">
+  <%
+  layer = 'ch.bav.seilbahnen-bundeskonzession.'
```

Comment:
Nice touch! Still, maybe just put it inline as it barely helps to increase readability or maintainability given there are only two instances or even only one further down.